### ISSUE TITLE
Multi-project/schema support

### ISSUE THREAD

**Issue by huysentruitw** [ChilliCream/graphql-platform#6205]
### Product

Strawberry Shake

### Is your feature request related to a problem?

We are using HotChocolate in our micro-services, but now, we are thinking to use StrawberryShake for service-to-service communication instead of gRPC or Web-API. In this setup, it is possible that one micro-service requires access to data in multiple other services, so we would like to be able to generate multiple clients (and use multiple graphql endpoints) in one micro-service (which is essentially one-csproj file).

Since each consuming-service has different query-requirements, we cannot create a single client project for every micro-service.

Slightly related: https://github.com/ChilliCream/graphql-platform/issues/4740#issuecomment-1039436780

### The solution you'd like

Would be great if StrawberryShake could work with multiple schemas.

The graphqlrc-file could already support that through the means of "projects", [see docs](https://the-guild.dev/graphql/config/docs/user/usage#projects).

So what I'd like is that we can configure something like this:

```graphql
{
  "projects": {
    "service-one": {
      "schema": "clients/service-one/schema.graphql",
      "documents": "clients/service-one/**/*.graphql",
      "extensions": {
        "strawberryShake": {
          "name": "ServiceOneClient",
          "namespace": "MyProject.Clients.ServiceOne",
          "url": "http://localhost:7000/graphql/",
          "records": {
            "inputs": false,
            "entities": false
          },
          "transportProfiles": [
            {
              "default": "Http",
              "subscription": "WebSocket"
            }
          ]
        }
      }
    },
    "service-two": {
      "schema": "clients/service-two/schema.graphql",
      "documents": "clients/service-two/**/*.graphql",
      "extensions": {
        "strawberryShake": {
          "name": "ServiceTwoClient",
          "namespace": "MyProject.Clients.ServiceTwo",
          "url": "http://localhost:8000/graphql/",
          "records": {
            "inputs": false,
            "entities": false
          },
          "transportProfiles": [
            {
              "default": "Http",
              "subscription": "WebSocket"
            }
          ]
        }
      }
    }
  }
}
```

**Comment by EvilVir** [comment]
I was able to do this by creating two config files in two separate directories, configured like this:

```
{
  "schema": "schema.graphql",
  "documents": "**/*.graphql",
  "extensions": {
    "strawberryShake": {
      "name": "ServiceA",
      "url": "http://localhost:15010/graphql",
      "dependencyInjection": true,
      "strictSchemaValidation": true,
      "hashAlgorithm": "md5",
      "useSingleFile": true,
      "requestStrategy": "Default",
      "outputDirectoryName": "Generated",
      "noStore": false,
      "emitGeneratedCode": true,
      "razorComponents": false,
      "records": {
        "inputs": false,
        "entities": false
      },
      "transportProfiles": [
        {
          "default": "Http",
          "subscription": "WebSocket"
        }
      ]
    }
  }
}
```

**Comment by huysentruitw** [comment]
@EvilVir thanks for your reply!

Pretty sure I tried this with StrawberryShake.Server v13.0.5, but let me try again with v13.1.0. 

Which package and version are you using in your project?

**Comment by huysentruitw** [comment]
Ok, seems to work. I will close this issue, as it solves the multi-schema issue I had. Thanks again.
